eZ/Publish/Core/Repository/SiteAccessAware/Tests/LocationServiceTest.php

```diff
@@ -61,12 +61,16 @@ public function providerForLanguagesLookupMethods()
         $contentInfo = new ContentInfo();
         $versionInfo = new VersionInfo();
 
-        // string $method, array $arguments, bool $return, int $languageArgumentIndex, ?callable $callback, ?int $alwaysAvailableArgumentIndex
+        // string $method, array $arguments, mixed|null $return, int $languageArgumentIndex, ?callable $callback, ?int $alwaysAvailableArgumentIndex
         return [
             ['loadLocation', [55], true, 1],
             ['loadLocation', [55, self::LANG_ARG], true, 1],
             ['loadLocation', [55, self::LANG_ARG, true], true, 1, null, 2],
 
+            ['loadLocationList', [[55]], ['return'], 1],
```

Comment:
As discussed internally, it would be better to mock here return of something more meaningful, like array of `Location` mock(s).

Reply:
fixed in f99d322